### ISSUE TITLE
Which repo is latest?

### ISSUE THREAD

**Issue by djmitche** [GothenburgBitFactory/GothenburgBitFactory.github.io#2]
https://github.com/GothenburgBitFactory/gbf.org and this repo appear to have very similar content, and similar modification dates. Which one is real?

**Comment by lauft** [comment]
@djmitche Previous to moving to GitHub pages, the repository contained an old version of the tasktools webpage (about 3dd2d489). It got out of date and unmaintained as the gbf.org repository was the one which got published via Netlify. 

With GitHub pages, repositories are published under `<organization>.github.io/<repository>` and the custom domain configured in the settings (e.g. https://gothenburgbitfactory.github.io/ti.net, which is rerouted to https://timewarrior.net).

For GitHub pages, the repository`<organization>.github.io` contains the default organization site, so it made sense for me to "move" gbf.org to here, instead of having it published under `gothenburgbitfactory.github.io/gbf.org` (it also sort of serves as a fallback for `gothenburgbitfactory.org`).

So, in short, both are real, this one is the one published, `gbf.org` is now the backup (which we now could delete, as the transfer to GitHub pages is successful and done).

**Comment by djmitche** [comment]
Thanks! Maybe it makes sense to archive (but not delete?) that repo, to avoid confusion.

**Comment by lauft** [comment]
I archived the old repo and [left a note](https://github.com/GothenburgBitFactory/gbf.org) there.
